Tests/js/views/assets/ez-universaldiscoveryview-tests.js

```diff
@@ -252,9 +301,206 @@ YUI.add('ez-universaldiscoveryview-tests', function (Y) {
         },
     });
 
+    renderTest = new Y.Test.Case({
+        name: "eZ Universal Discovery View render test",
+
+        setUp: function () {
+            this.title = 'Universal discovery view title';
+            this.selectionMode = 'multiple';
+            this.method1 = new Y.eZ.UniversalDiscoveryMethodBaseView({
+                identifier: 'method1',
+            });
+            this.method2 = new Y.eZ.UniversalDiscoveryMethodBaseView({
+                identifier: 'method2',
+            });
+            this.view = new Y.eZ.UniversalDiscoveryView({
+                container: '.container',
+                title: this.title,
+                visibleMethod: 'method2',
+                selectionMode: this.selectionMode,
+                methods: [this.method1, this.method2],
+            });
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            this.method1.destroy();
+            this.method2.destroy();
+            delete this.view;
+            delete this.method1;
+            delete this.method2;
+        },
+
+        "Should initialize function visibility of the method views": function () {
+            Assert.isTrue(
+                this.method2.get('visible'),
+                "The method2 should be visible"
+            );
+            Assert.isFalse(
+                this.method1.get('visible'),
+                "The method1 should not be visible"
+            );
+        },
+
+        "Should change the visible flag depending on visibleMethod": function () {
+            this.view.set('visibleMethod', 'method1');
+            Assert.isTrue(
+                this.method1.get('visible'),
+                "The method1 should be visible"
+            );
+            Assert.isFalse(
+                this.method2.get('visible'),
+                "The method2 should not be visible"
+            );
+        },
+    });
+
+    tabTest = new Y.Test.Case({
+        name: "eZ Universal Discovery View render test",
+
+        setUp: function () {
+            this.method1 = new Y.eZ.UniversalDiscoveryMethodBaseView();
+            this.method1._set("title", "Method 1");
```

Comment:
Why don't set `title` and `identifier` when initializing `Y.eZ.UniversalDiscoveryMethodBaseView`?
```
new Y.eZ.UniversalDiscoveryMethodBaseView({
  title: 'Method 1',
  identifier: 'method1'
});
```